### ISSUE TITLE
Broken Travis builds

### ISSUE THREAD

**Issue by guygriffiths** [Reading-eScience-Centre/ncwms#18]
Travis is building on every push to `develop`, and is failing due to the fact that none of the EDAL libraries are available (since we don't publish EDAL SNAPSHOT dependencies anywhere).

@yosoyjay - any idea how we can disable Travis builds of the `develop` branch?

**Comment by yosoyjay** [comment]
It's specified in the `.travis.yml` file.  Do you want to disable only for the `develop` branch, or only test for `master`?

**Comment by guygriffiths** [comment]
I'd initially thought we should only have Travis build on pushes to `master` (i.e. only on release), but after performing a release yesterday I've disabled it entirely - since it takes less time to do all of the stages of a release than it does for the EDAL libraries to filter through to Maven Central, the build still fails.

I guess the ideal would be to only build on pushes to `master`, but to leave a few hours delay before attempting to build?  But perhaps that defeats the point of CI?

**Comment by yosoyjay** [comment]
Yeah, that defeats the purpose of the continuous integration.  You want to catch problems as soon as possible.

Would it instead be feasible to publish the EDAL SNAPSHOTs?

How did you disable the testing?  I still see the `.travis.yml` file in both the `master` and `develop` branches.

**Comment by guygriffiths** [comment]
It's fairly rare that we change anything substantial in ncWMS2 - the vast majority of code changes are in EDAL.  The only reason this was triggered this time was because I updated the documentation.

I could try to publish the EDAL SNAPSHOTS when making a significant change to `develop` but we still have the issue that we'll need to leave a significant delay between publishing EDAL releases and ncWMS2 releases.

I disabled it on the Travis website, there was a toggle switch for each of our GitHub repos.
